packages/components/document-card/src/document-card.js

```diff
@@ -30,10 +34,27 @@ customElementDefineHelper(
 			return this.dir || this.bodyDir;
 		}
 
-		get fileIcon() {
+		get icon() {
+			let icon;
+			switch (this.cardType) {
```

Comment:
Can you please instead of switch use an object?

const icons = {
 marketplace: marketplaceIcon, 
...
}

icons[this.type]

Reply:
yeah, but it does require a default too

Reply:
Default can be also have key in icons:
icons[this.type] || icons[constants.ICONS_DEFAULT]